examples/slice_explorer.py

```diff
@@ -0,0 +1,24 @@
+from argparse import ArgumentParser, BooleanOptionalAction
+
+from pan3d import DatasetBuilder
+from pan3d import SliceExplorer
+
+def serve():
+    parser = ArgumentParser(
```

Comment:
Technically you already have a parser on `server.cli`

Reply:
This example exists outside of `Trame` context and the server is built inside the `SliceExplorer` instance.